background/services/provider-bridge/index.ts

```diff
@@ -0,0 +1,123 @@
+import browser from "webextension-polyfill"
+import {
+  ChainService,
+  ServiceCreatorFunction,
+  ServiceLifecycleEvents,
+} from ".."
+import logger from "../../lib/logger"
+import BaseService from "../base"
+import InternalEthereumProviderService from "../internal-ethereum-provider"
+import { getOrCreateDB, ProviderBridgeServiceDatabase } from "./db"
+
+type Events = ServiceLifecycleEvents & {
+  accountAccessRequestedByDAppID: string
+}
+
+/**
+ * The ProviderBridgeService is responsible for the communication with the
+ * provider-bridge (content-script).
+ *
+ * The main purpose for this service/layer is to provide a transition
+ * between the untrusted communiction from the window-provider - which runs
+ * in user space and can be modifed by many attacks - and our internal service
+ * layer.
```

Comment:
```suggestion
 * in shared dapp space and can be modified by other extensions - and our
 * internal service layer.
```

Changing “user” to “dApp” because the mental model is a little different than the usual userspace/kernel space distinction. Does this seem like a fair rephrasing?